### DISCUSSION TITLE
Is it possible to take a conjugated verb?

### DISCUSSION THREAD

**Discussion by gleidsonh** [explosion/spaCy#6360] [Help: Coding & Implementations] [answered]
Let's suppose we have the phrase: "Yesterday I bought a car"

Using Spacy Wordnet, I can get a list of synonyms, but it is lemmatized. (buy | purchase)

I need to go through this list, and based on the morphology of the word "bought", I will find the synonyms in the same tense.

How to do this?

**Comment by adrianeboyd** [comment, marked as answer]
spacy is mainly focused on analysis rather than generation, so the core library doesn't include any components that generate forms, for example generating the inflected form based on a lemma + morphological features. You'll need to look for alternate libraries with support for morphological generation for English to do this.
